apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/components/ResponseFilter.tsx

```
@@ -224,63 +226,66 @@ export const ResponseFilter = ({ survey }: ResponseFilterProps) => {
-                />
-              </div>
-            </div>
-            {i !== filterValue.filter.length - 1 && (
-              <div className="my-6 flex items-center">
-                <p className="mr-6 text-base text-slate-600">And</p>
-                <hr className="w-full text-slate-600" />
+
+        <div ref={parent}>
+          {filterValue.filter?.map((s, i) => (
+            <>
+              <div className="flex w-full flex-wrap gap-3 md:flex-nowrap">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)